packages/http/src/util/runtimeExpression.ts

```diff
@@ -0,0 +1,128 @@
+import { IHttpRequest, IHttpResponse } from '../types';
+import * as Option from 'fp-ts/lib/Option';
+import { lookup } from 'fp-ts/lib/Array';
+import { pipe } from 'fp-ts/lib/pipeable';
+import { get as _get } from 'lodash';
```

Comment:
Why the renaming?

Reply:
`get` is too common. However, I'm good with any approach. We could agree on `get`, `_get` or `_.get`.

Reply:
We've bee using `get` everywhere here in Prism. It's not a big deal though, just mentioning for consistency.